### DISCUSSION TITLE
Where are the release notes for 1.3.3 (1.3.230724000)?

### DISCUSSION THREAD

**Discussion by harvinders** [microsoft/WindowsAppSDK#3785] [Q&A] [answered]
I see that there is a [nuget package](https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.3.230724000) available but there are no release notes to be found anywhere.

**Comment by Mysterious-Dev** [comment, marked as answer]
https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/stable-channel#version-133-13230724000
